buildenv

```diff
@@ -37,15 +38,22 @@ zopen_post_install() {
 
 zopen_check_results()
 {
-  dir="$1"
-  pfx="$2"
-  chk="$1/$2_check.log"
-
-  # Echo the following information to gauge build health
-  echo "actualFailures:0"
-  echo "totalTests:1"
-  echo "expectedFailures:0"
-  echo "expectedTotalTests:1"
+  chk="./test_run.log"
+
+  # Count actual errors and passes based on log patterns
+  actualErrors=$(grep -o "EXCEPTION IN: " "$chk" | wc -l)
+  actualPass=$(grep -o "LOG FOR: " "$chk" | wc -l)
+
+  # Calculate total tests and expected failures
+  totalTests=$((actualPass + actualErrors))
+  expectedErrors="2"
```

Comment:
```suggestion
  expectedErrors="2"
```
Should expected errors be 2 or should it match the # we get in the public dev system?